barretenberg/cpp/src/barretenberg/vm2/simulation/events/execution_event.hpp

```diff
@@ -21,6 +22,10 @@ struct ExecutionEvent {
     Instruction wire_instruction;
     ExecutionOpCode opcode;
     std::vector<Operand> resolved_operands;
+    // Inputs and Outputs are used in register allocation in tracegen
```

Comment:
Maybe say in the comments that these are inputs and outputs to the gadget/opcode.